app/helpers/Intersect.d.ts

```diff
@@ -1,7 +1,8 @@
 import { checkIntersect as intersect } from './Intersect';
 import { HealthcareAuthority } from '../store/types';
+import { ExposureDatum } from '../exposureHistory';
 
 export const checkIntersect: (
   healthcareAuthorities: HealthcareAuthority[] | null,
   bypassTimer: boolean,
-) => Promise<void> = intersect;
+) => Promise<Record<number, ExposureDatum>> = intersect;
```

Comment:
```suggestion
type Posix = number
 
 ....
 
 
) => Promise<Record<Posix, ExposureDatum>> = intersect;
```

its helpful context for the reader to indicate that the number represents a posix value

Reply:
Agree, I'll add it now.